rest_framework/authtoken/serializers.py

```diff
@@ -18,13 +19,25 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _('User account is disabled.')
-                    raise serializers.ValidationError(msg)
+                    raise serializers.ValidationError(
+                        ValidationErrorMessage(
+                            msg,
+                            code='authorization')
```

Comment:
This is just me not quite mentally parsing that this change
modify the existing `message` argument, rather than adding a new, separate, optional `code` argument. Apologies for continuing to be a thorn in this, but can't say I'm terribly keen on this API. I'd rather see us eg mirror Django here... https://docs.djangoproject.com/en/1.9/_modules/django/core/exceptions/#ValidationError